docs/developers-clear-containers-install.md

```diff
@@ -78,6 +78,9 @@ See [the upgrading document](upgrading.md) for further details.
    $ sudo make install
    ```
 
+   Note: the install step above will overwrite any proxy binary installed from
```

Comment:
Side note - so we overwrite the shim and proxy, but not the runtime - and I think that is because the runtime build system has a notion of 'dev build' vs 'package build', but the others do not. OK, that's good to document, and it seems a bit late and un-necessary to fix that right now - but, in the future let's see if we can remember to probably have dev and package builds for all the components :-)

Reply:
Yes, that's it exactly. And yes, let's all try to remember to make this a hard requirement for kata! ;)